region/maridia/outer/Fish Tank.json

```diff
@@ -359,6 +359,17 @@
       ],
       "note": "Once high enough, it may be necessary to kill the fish and open the door."
     },
+    {
+      "link": [1, 4],
+      "name": "Gravity Jump",
+      "requires": [
+        "canGravityJump",
+        {"or": [
+          "HiJump",
+          "canCarefulJump"
```

Comment:
This might not be obviously possible in medium?  I don't know what the reaction is to a decentish looking gravity jump that doesn't reach the door.
I wouldn't move its placement or make a notable.  So that must mean I'd leave it like this.

Reply:
There arent many that require this big of a jump, those that are more than it I think require a `canTrickyJump`. The window didn't feel too tight compared to other `canCarefulJump` strats and you can retry it.